projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -341,3 +341,31 @@ export class CollectionEffects {
 </div>
 
 To learn about testing effects that incorporate state, see the [Effects that use State](guide/effects/testing#effect-that-uses-state) section in the testing guide.
+
+## Non-Action Effects
+
+Because all effects are basically RxJS `Observable`s, they can be used without actions and the `ofType` operator. This is useful for effects that don't need to listen to some specific actions, but rather to some other `Observable`s from our application. For example. imagine we want to track user activity (clicks anywhere, for example), and send that data to our monitoring server. We can do this by creating a custom effect that listens to the `document` `click` event and emits the event data to our server:
```

Comment:
```suggestion
Because effects are merely consumers of observables, they can be used without actions and the `ofType` operator. This is useful for effects that don't need to listen to some specific actions, but rather to some other observable source. For example, imagine we want to track whenever click events happen and send that data to our monitoring server. This can be done by creating a custom effect that listens to the `document` `click` event and emits the event data to our server:
```